### ISSUE TITLE
Calendar title overflows inside calendar dropdown

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1741]
### Description

Inside the Create/Edit Event modal, in the calendar dropdown, calendars with long titles overflow beyond the dropdown boundaries.
![Screenshot 2024-12-16 131524](https://github.com/user-attachments/assets/7cdc9b1a-8c14-490e-9add-4e46faf41acf)

### Steps to reproduce

1. Create a calendar with a very long title
2. Open the create/edit modal and change the calendar

### Expected behavior

Calendars with long titles should be displayed within the boundaries of the dropdown menu without causing layout issues.

### Which operating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Mozilla Firefox

### Extensions

.

### Version

2024.9.1
